plugins/lando-app/tasks/share.js

```diff
@@ -49,7 +49,7 @@ module.exports = lando => {
           // Get the URLS
           .then(() => {
             // Assume a port to start
-            const port = 80;
+            var port = 80;
```

Comment:
we should use let instead of var here.

Reply:
yes, `let` is now the preferred syntax in our es6 things